indexer/services/comlink/src/controllers/api/v4/asset-positions-controller.ts

```diff
@@ -159,11 +161,14 @@ router.get(
       subaccountNumber,
     }: AssetPositionRequest = matchedData(req) as AssetPositionRequest;
 
+    // The schema checks allow subaccountNumber to be a string, but we know it's a number here.
```

Comment:
Doesn't `subaccountNumber` already have the type `number`?

Reply:
Yes, but that only applies to compile time checks. Someone can still pass in a string and the schema checks let that through

Reply:
yea, makes sense, I see that there's no explicit conversion to `number` only the `as` operator that tells typescript to treat the data as an `AssetPositionRequest`.